src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1362,6 +1362,7 @@ namespace winrt::TerminalApp::implementation
         return connection;
     }
 
+    // use _replaceConnectionForRestart instead of using this function directly. _replaceConnectionForRestart will close the old connection.
     TerminalConnection::ITerminalConnection TerminalPage::_duplicateConnectionForRestart(const TerminalApp::TerminalPaneContent& paneContent)
```

Comment:
Hmm... If we shouldn't use `_duplicateConnectionForRestart` and there's only one place we're using it (in `_replaceConnectionForRestart`), can we just combine them so nobody uses the wrong one in the future?